pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerDescribeStreamCommand.java

```diff
@@ -82,53 +76,33 @@ public void execute() {
 
             // Output the configuration of this Stream.
             CompletableFuture<StreamConfiguration> streamConfig = store.getConfiguration(scope, stream, null, executor);
-            responseBuilder.append("Stream configuration: ").append(streamConfig.join().toString()).append("\n");
+            prettyJSONOutput("stream config", streamConfig.join());
```

Comment:
I'd try to avoid spaces and special characters in JSON keys. Alternatively you can use underscore or camelCase formatting.

Reply:
I replaces spaces by underscores in all the JSON keys.